docs/operators/droplet-quickstart.md

```diff
@@ -0,0 +1,240 @@
+# Quickstart for running nwaku on a DigitalOcean Droplet
+
+This guide explains how to run a nwaku node on a 
+DigitalOcean Droplet. We enable the following protocols -
+
+1. Relay
+2. Store
+3. DNS Discovery
+4. Discv5
+
+A Droplet is a simple virtual machine that runs in DigitalOcean's datacenters.
+
+Note that Droplets do cost money, the size described in the guide costs approximately $12 a month. 
+
+The guide makes heavy use of the `doctl` cli to make it as UI agnostic as possible. 
+There are similar steps to accomplish the same through DigitalOcean's cloud console, accessible [here](https://cloud.digitalocean.com/)
+
+## Prerequisites
+
+1. A DigitalOcean account. Upon signing up, you have $100 worth of credits to use.
+
+
+
+## 1. Get the `doctl` binary
+
+Follow this [guide](https://docs.digitalocean.com/reference/doctl/how-to/install/) to install,
```

Comment:
Perhaps a
> Note

here to say that it's not necessary to follow the droplet creation steps in the `doctl` guide as we'll set up a new droplet below.

Reply:
Addressed in [3febd72](https://github.com/status-im/nwaku/pull/1133/commits/3febd72c0923543883064290187a999d3f291c4e)